boards/redboard_redv/src/main.rs

```diff
@@ -247,10 +268,7 @@ pub unsafe fn main() {
     )
     .finalize(());
 
-    // Need two debug!() calls to actually test with QEMU. QEMU seems to have
-    // a much larger UART TX buffer (or it transmits faster).
-    debug!("Red-V initialization complete.");
-    debug!("Entering main loop.");
```

Comment:
Why not leave this as is?

Reply:
Because I couldn't reproduce the bug and have tried many times to do so.

Reply:
Is there a reason not to just leave it though?

Reply:
IIRC, the bug was not "without two lines nothing prints" but rather: "with a single debug!() statement, things will seem to work in QEMU even if there is a breakage". Basically, because QEMU has a large buffer / transmits quickly, the entire message is printed to console even if the kernel loop does not run. So without a two line message here, our QEMU CI will fail to catch basic breakages. Now, we don't even run the redboard_redv in QEMU as part of CI, so it may not matter much here, but I seem to recall that being the justification for this when it was first added for the hifive1.

Reply:
I understand your point of view. I will leave it as it is and add it back to the HiFive1 with a more in depth explanation.